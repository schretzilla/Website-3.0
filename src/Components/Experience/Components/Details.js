import React from "react";

import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";

const Details = props => {
  const { experienceData } = props;
  const { company, title, dates, description, teams, keyPoints, link, logo } = experienceData;

  const style = {
    logo: {
      width: "327px",
    },
    container: {
      paddingBottom: "50px",
    },
  };
  const keyPointElement = keyPoints.map(keyPoint => (
    <li key={`key-point-${company}-${title}`} className="job-key-points">
      {keyPoint}
    </li>
  ));

  const linkToExtraData = (
    <a target="_blank" href={link.address} rel="noopener noreferrer">
      {link.name}
    </a>
  );

  return (
    <div style={style.container}>
      <Row>
        <Col md={6}>
          <img src={logo} style={style.logo} alt={`${company}-logo`} />
          <h2 className="job-title">{title}</h2>
        </Col>
        <Col className="subject-sub-date" md={6}>
          <p className="float-md-right">{dates}</p>
        </Col>
      </Row>

      <Row>
        <Col sm={12}>
          <p className="job-description">
            {description} {linkToExtraData}{" "}
          </p>
          <p className="job-team-name">{teams}</p>
        </Col>

        <ul>{keyPointElement} </ul>
      </Row>
    </div>
  );
};

export default Details;
