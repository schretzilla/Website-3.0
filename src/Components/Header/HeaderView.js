import React from "react";

import Container from "react-bootstrap/Container";

const Header = () => (
  <div className="parallax">
    <Container>
      <div className="vertical-center jumbotron">
        <h1 className="name-header">Scott Schretzenmaier</h1>
        <p id="name-description">Software Engineer, Bug Squasher, Adventure Seeker</p>
      </div>
    </Container>
  </div>
);

export default Header;
