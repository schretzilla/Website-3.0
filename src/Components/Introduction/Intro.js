import React from "react";

import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";

import profilePhoto from "../../Images/Introduction/Profile-Photo.png";
import githubIcon from "../../Images/Introduction/github-icon.png";
import linkedInIcon from "../../Images/Introduction/linkedin-logo.png";
import resumeIcon from "../../Images/Introduction/resume-icon.png";

import Jump from "react-reveal/Jump";

const Intro = props => {
  const style = {
    introContainer: {
      marginTop: "40px",
      marginBottom: "50px"
    },
    icon: {
      height: "37px",
      margin: "5px"
    }
  };
  return (
    <Row style={style.introContainer}>
      <Col sm={12} md={6}>
        <img
          className="img-fluid"
          src={profilePhoto}
          alt="scott-schretzenmaier-profile"
        />
      </Col>
      <Col sm={12} md={6}>
        <Jump delay={1000}>
          <h1 className="hello-header">Hi, I'm Scott</h1>
        </Jump>
        <p className="about-me">
          I'm a motivated guy who loves building awesome defense software. I
          prefer backend development but I dabble with a bit of everything. I'm
          also a big fan of agile processes, both in development and in life.
        </p>

        <ul className="list-inline">
          <li>
            <strong>Email:</strong> schretzy929@gmail.com
          </li>
          <li>
            <strong>Location:</strong> Los Angeles, CA
          </li>
        </ul>

        <div>
          <a
            href="https://github.com/schretzilla"
            target="_blank"
            rel="noopener noreferrer"
          >
            <img style={style.icon} src={githubIcon} alt="github logo" />
          </a>

          <a
            href="http://www.linkedin.com/in/scott-schretzenmaier-26ab1178"
            target="_blank"
            rel="noopener noreferrer"
          >
            <img style={style.icon} src={linkedInIcon} alt="Linked In logo" />
          </a>
          <a
            href="https://drive.google.com/open?id=1UzKlP_4Am8XAe6fSScxsFzNZIMKlkeUE"
            target="_blank"
            rel="noopener noreferrer"
          >
            <img style={style.icon} src={resumeIcon} alt="Resume" />
          </a>
        </div>
      </Col>
    </Row>
  );
};

export default Intro;
