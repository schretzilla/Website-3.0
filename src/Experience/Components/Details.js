import React from 'react';

import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';

import '../../style/custom.css';

const Details = props => {
  const { experienceData } = props;
  const {
    company,
    title,
    dates,
    description,
    teams,
    keyPoints,
    link
  } = experienceData;

  const keyPointElement = keyPoints.map((keyPoint, i) => {
    return (
      <li key={`key-point-${company}-${i}`} className="job-key-points">
        {keyPoint}
      </li>
    );
  });

  const linkToExtraData = (
    <a target="_blank" href={link.address} rel="noopener noreferrer">
      {link.name}
    </a>
  );

  return (
    <>
      <Row>
        <Col sm={8}>
          <h2 className="company-name">{company}</h2>
          <h2 className="job-title">{title}</h2>
        </Col>
        <Col className="subject-sub-date" sm={4}>
          <h2>{dates}</h2>
        </Col>
      </Row>

      <Row>
        <Col sm={12}>
          <p className="job-description">
            {description} {linkToExtraData}{' '}
          </p>
          <p className="job-team-name">{teams}</p>
        </Col>

        <ul>{keyPointElement} </ul>
      </Row>
      <hr className="featurette-divider" />
    </>
  );
};

export default Details;
