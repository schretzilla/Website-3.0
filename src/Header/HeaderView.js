import React from 'react';

import Container from 'react-bootstrap/Container';

import '../style/custom.css';

const Header = () => {
  return (
    <>
      <div className="parallax">
        <Container>
          <div className="vertical-center jumbotron">
            <h1 className="name-header">Scott Schretzenmaier</h1>
            <p id="name-description">
              Software Engineer, Bug Squasher, Adventure Seeker
            </p>
          </div>
        </Container>
      </div>
    </>
  );
};

export default Header;
