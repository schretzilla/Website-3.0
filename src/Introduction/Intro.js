import React from "react";

import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";

import profilePhoto from "../Images/Profile-Photo.png";
import githubIcon from "../Images/github-icon.png";
import linkedInIcon from "../Images/linkedin-logo.png";
import resumeIcon from "../Images/resume-icon.png";

import "../style/custom.css";

const Intro = props => {
  const style = {
    introContainer: {
      marginTop: "40px"
    },
    icon: {
      height: "37px",
      margin: "5px"
    }
  };
  return (
    <>
      <Row style={style.introContainer}>
        <Col sm={12} md={6}>
          <img
            className="img-fluid"
            src={profilePhoto}
            alt="scott-schretzenmaier-profile"
          />
        </Col>
        <Col sm={12} md={6}>
          <h1 className="hello-header">Hi, I'm Scott</h1>
          <p className="about-me">
            I'm a motivated guy who loves building awesome defense software. I
            prefer backend development but I dabble with a bit of everything.
            I'm also a big fan of agile processes, both in development and in
            life.
          </p>

          <ul className="list-inline">
            <li>
              <strong>Email:</strong> schretzy929@gmail.com
            </li>
            <li>
              <strong>Location:</strong> Los Angeles, CA
            </li>
          </ul>

          <a
            href="https://github.com/schretzilla"
            target="_blank"
            rel="noopener noreferrer"
          >
            <img style={style.icon} src={githubIcon} alt="github logo" />
          </a>

          <a
            href="http://www.linkedin.com/in/scott-schretzenmaier-26ab1178"
            target="_blank"
            rel="noopener noreferrer"
          >
            <img style={style.icon} src={linkedInIcon} alt="Linked In logo" />
          </a>
          <a
            href="https://drive.google.com/open?id=1UzKlP_4Am8XAe6fSScxsFzNZIMKlkeUE"
            target="_blank"
            rel="noopener noreferrer"
          >
            <img style={style.icon} src={resumeIcon} alt="Resume" />
          </a>
        </Col>
      </Row>
    </>
  );
};

export default Intro;
