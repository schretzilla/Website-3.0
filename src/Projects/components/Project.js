import React from "react";

import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";

import "../../style/custom.css";

const Project = props => {
  const { projectData, indexNumber } = props;

  const extraDetailsList = projectData.extraDetails.map((detail, i) => {
    return <li key={`${projectData.name}-detail-${i}`}>{detail}</li>;
  });

  const toolsList = projectData.tools.map((tool, i) => {
    return (
      <li className="list-inline-item" key={`${projectData.name}-tool-${i}`}>
        {tool}
      </li>
    );
  });

  const teamMemberList = projectData.team.map((member, i) => {
    return (
      <li className="list-inline-item" key={`${projectData.name}-member-${i}`}>
        {member}
      </li>
    );
  });

  const imageColumn = (
    <Col sm={12} md={5}>
      Image goes here
    </Col>
  );

  const descriptionColumn = (
    <Col sm={12} md={7}>
      <h2 className="featurette-heading">
        {projectData.name}&nbsp;
        <small className="text-muted">{projectData.catchPhrase}</small>
      </h2>
      <p className="lead">{projectData.blurb}</p>

      <ol>{extraDetailsList}</ol>

      <ul className="list-inline">
        <li className="list-inline-item">
          <strong>Tools:</strong>
        </li>
        {toolsList}
      </ul>

      <ul className="list-inline">
        <li className="list-inline-item">
          <strong>Team:</strong>
        </li>
        {teamMemberList}
      </ul>

      <h5 className="text-left">
        <small className="text-muted">{projectData.date}</small>
      </h5>
    </Col>
  );

  let projectContent;
  if (indexNumber % 2 === 0) {
    projectContent = (
      <>
        {imageColumn} {descriptionColumn}
      </>
    );
  } else {
    projectContent = (
      <>
        {descriptionColumn} {imageColumn}
      </>
    );
  }
  return (
    <>
      <Row>{projectContent}</Row>

      <hr className="featurette-divider" />
    </>
  );
};

export default Project;
