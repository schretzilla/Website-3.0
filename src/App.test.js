import React from "react";
import renderer from "react-test-renderer";

import Header from "./Components/Header/HeaderView";
import Intro from "./Components/Introduction/Intro";
import Details from "./Components/Experience/Components/Details";
import Project from "./Components/Projects/components/Project";

import trailCamera from "./Images/Projects/trail-camera-placement.jpeg";
import HikingLogo from "./Images/Experience/gone-hiking-logo.png";

describe("the Header component", () => {
  it("snapshot test", () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("the Intro component", () => {
  it("snapshot test", () => {
    const tree = renderer.create(<Intro />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("the Experience component", () => {
  const testExpData = {
    company: "Cross Country Traveler",
    logo: HikingLogo,
    title: "Solo Adventurer",
    dates: "August/2016-January/2017",
    teams: "",
    description: "A solo 6 month adventure across America summarized using D3",
    keyPoints: [
      "Volunteered with Samaritain's Purse helping Louisiana flood victims",
      "Summited 24 of Coloradoâ€™s 14,000+ foot mountains",
      "Confirmed strategy and required actions for my five-year plan"
    ],
    link: {
      address: "https://schretzilla.github.io/adventure-america/index",
      name: "here"
    }
  };
  it("snapshot test with link and image logo", () => {
    const tree = renderer
      .create(<Details experienceData={testExpData} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("the Projects component", () => {
  const projectData = {
    name: "Smart Bird Feeder!",
    catchPhrase: "Building species saving technology",
    blurb:
      "I had the opportunity to lead a team of engineers as we worked with the San Diego Zoo to develop a Smart Bird Feeder that monitors the health of the Hawaiian Crows. At the conclusion of a month the device we created included the following:",
    extraDetails: [
      "A bird feeder that includes a variety of sensors to track the diet of each bird",
      "A repurposed android cell phone that records video of the crows as they feed",
      "Remote login capabilities to watch from the bird feeder at any time",
      "A user interface for tracking each birdâ€™s status"
    ],
    tools: ["Android SDK", "Java", "A 3D Enclosure", "Teensy 3.2"],
    team: ["Avery Bodenstein", "Andrew Miyaguchi", "John Madden"],
    date: "July/2019",
    media: {
      type: "image",
      name: "trail camera 3-D model",
      src: trailCamera
    }
  };
  describe("snapshot tests", () => {
    it("image on right", () => {
      const tree = renderer
        .create(<Project projectData={projectData} indexNumber="1" />)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
    it("image on left", () => {
      const tree = renderer
        .create(<Project projectData={projectData} indexNumber="2" />)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
