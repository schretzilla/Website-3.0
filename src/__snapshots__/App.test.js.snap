// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Experience component snapshot test with link and image logo 1`] = `
<div
  style={
    Object {
      "paddingBottom": "50px",
    }
  }
>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <img
        alt="Cross Country Traveler-logo"
        src="gone-hiking-logo.png"
        style={
          Object {
            "width": "327px",
          }
        }
      />
      <h2
        className="job-title"
      >
        Solo Adventurer
      </h2>
    </div>
    <div
      className="subject-sub-date col-md-6"
    >
      <p
        className="float-md-right"
      >
        August/2016-January/2017
      </p>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-sm-12"
    >
      <p
        className="job-description"
      >
        A solo 6 month adventure across America summarized using D3
         
        <a
          href="https://schretzilla.github.io/adventure-america/index"
          rel="noopener noreferrer"
          target="_blank"
        >
          here
        </a>
         
      </p>
      <p
        className="job-team-name"
      >
        
      </p>
    </div>
    <ul>
      <li
        className="job-key-points"
      >
        Volunteered with Samaritain's Purse helping Louisiana flood victims
      </li>
      <li
        className="job-key-points"
      >
        Summited 24 of Colorado’s 14,000+ foot mountains
      </li>
      <li
        className="job-key-points"
      >
        Confirmed strategy and required actions for my five-year plan
      </li>
       
    </ul>
  </div>
</div>
`;

exports[`the Header component snapshot test 1`] = `
<div
  className="parallax"
>
  <div
    className="container"
  >
    <div
      className="vertical-center jumbotron"
    >
      <h1
        className="name-header"
      >
        Scott Schretzenmaier
      </h1>
      <p
        id="name-description"
      >
        Software Engineer, Bug Squasher, Adventure Seeker
      </p>
    </div>
  </div>
</div>
`;

exports[`the Intro component snapshot test 1`] = `
<div
  className="row"
  style={
    Object {
      "marginBottom": "50px",
      "marginTop": "40px",
    }
  }
>
  <div
    className="col-md-6 col-sm-12"
  >
    <img
      alt="scott-schretzenmaier-profile"
      className="img-fluid"
      src="Profile-Photo.png"
    />
  </div>
  <div
    className="col-md-6 col-sm-12"
  >
    <h1
      className=" hello-header"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      Hi, I'm Scott
    </h1>
    <p
      className="about-me"
    >
      I'm a motivated guy who loves building awesome defense software. I prefer backend development but I dabble with a bit of everything. I'm also a big fan of agile processes, both in development and in life.
    </p>
    <ul
      className="list-inline"
    >
      <li>
        <strong>
          Email:
        </strong>
         schretzy929@gmail.com
      </li>
      <li>
        <strong>
          Location:
        </strong>
         Los Angeles, CA
      </li>
    </ul>
    <div>
      <a
        href="https://github.com/schretzilla"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="github logo"
          src="github-icon.png"
          style={
            Object {
              "height": "37px",
              "margin": "5px",
            }
          }
        />
      </a>
      <a
        href="http://www.linkedin.com/in/scott-schretzenmaier-26ab1178"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Linked In logo"
          src="linkedin-logo.png"
          style={
            Object {
              "height": "37px",
              "margin": "5px",
            }
          }
        />
      </a>
      <a
        href="https://drive.google.com/open?id=1UzKlP_4Am8XAe6fSScxsFzNZIMKlkeUE"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Resume"
          src="resume-icon.png"
          style={
            Object {
              "height": "37px",
              "margin": "5px",
            }
          }
        />
      </a>
    </div>
  </div>
</div>
`;

exports[`the Projects component snapshot tests image on left 1`] = `
Array [
  <div
    className="row"
  >
    <div
      className="col-md-5 col-12 order-md-0 order-1"
    >
      <img
        alt="trail camera 3-D model"
        className="img-fluid"
        src="trail-camera-placement.jpeg"
      />
    </div>
     
    <div
      className="col-md-7 col-sm-12"
    >
      <h2
        className="featurette-heading"
      >
        Smart Bird Feeder!
         
        <small
          className="text-muted"
        >
          Building species saving technology
        </small>
      </h2>
      <p
        className="lead"
      >
        I had the opportunity to lead a team of engineers as we worked with the San Diego Zoo to develop a Smart Bird Feeder that monitors the health of the Hawaiian Crows. At the conclusion of a month the device we created included the following:
      </p>
      <ol>
        <li>
          A bird feeder that includes a variety of sensors to track the diet of each bird
        </li>
        <li>
          A repurposed android cell phone that records video of the crows as they feed
        </li>
        <li>
          Remote login capabilities to watch from the bird feeder at any time
        </li>
        <li>
          A user interface for tracking each bird’s status
        </li>
      </ol>
      <ul
        className="list-inline"
      >
        <li
          className="list-inline-item"
        >
          <strong>
            Tools:
          </strong>
        </li>
        <li
          className="list-inline-item"
        >
          Android SDK
        </li>
        <li
          className="list-inline-item"
        >
          Java
        </li>
        <li
          className="list-inline-item"
        >
          A 3D Enclosure
        </li>
        <li
          className="list-inline-item"
        >
          Teensy 3.2
        </li>
      </ul>
      <ul
        className="list-inline"
      >
        <li
          className="list-inline-item"
        >
          <strong>
            Team:
          </strong>
        </li>
        <li
          className="list-inline-item"
        >
          Avery Bodenstein
        </li>
        <li
          className="list-inline-item"
        >
          Andrew Miyaguchi
        </li>
        <li
          className="list-inline-item"
        >
          John Madden
        </li>
      </ul>
      <h5
        className="text-left"
      >
        <small
          className="text-muted"
        >
          July/2019
        </small>
      </h5>
    </div>
  </div>,
  <hr
    className="featurette-divider"
  />,
]
`;

exports[`the Projects component snapshot tests image on right 1`] = `
Array [
  <div
    className="row"
  >
    <div
      className="col-md-7 col-sm-12"
    >
      <h2
        className="featurette-heading"
      >
        Smart Bird Feeder!
         
        <small
          className="text-muted"
        >
          Building species saving technology
        </small>
      </h2>
      <p
        className="lead"
      >
        I had the opportunity to lead a team of engineers as we worked with the San Diego Zoo to develop a Smart Bird Feeder that monitors the health of the Hawaiian Crows. At the conclusion of a month the device we created included the following:
      </p>
      <ol>
        <li>
          A bird feeder that includes a variety of sensors to track the diet of each bird
        </li>
        <li>
          A repurposed android cell phone that records video of the crows as they feed
        </li>
        <li>
          Remote login capabilities to watch from the bird feeder at any time
        </li>
        <li>
          A user interface for tracking each bird’s status
        </li>
      </ol>
      <ul
        className="list-inline"
      >
        <li
          className="list-inline-item"
        >
          <strong>
            Tools:
          </strong>
        </li>
        <li
          className="list-inline-item"
        >
          Android SDK
        </li>
        <li
          className="list-inline-item"
        >
          Java
        </li>
        <li
          className="list-inline-item"
        >
          A 3D Enclosure
        </li>
        <li
          className="list-inline-item"
        >
          Teensy 3.2
        </li>
      </ul>
      <ul
        className="list-inline"
      >
        <li
          className="list-inline-item"
        >
          <strong>
            Team:
          </strong>
        </li>
        <li
          className="list-inline-item"
        >
          Avery Bodenstein
        </li>
        <li
          className="list-inline-item"
        >
          Andrew Miyaguchi
        </li>
        <li
          className="list-inline-item"
        >
          John Madden
        </li>
      </ul>
      <h5
        className="text-left"
      >
        <small
          className="text-muted"
        >
          July/2019
        </small>
      </h5>
    </div>
     
    <div
      className="col-md-5 col-12 order-md-0 order-1"
    >
      <img
        alt="trail camera 3-D model"
        className="img-fluid"
        src="trail-camera-placement.jpeg"
      />
    </div>
  </div>,
  <hr
    className="featurette-divider"
  />,
]
`;

exports[`the Tech Stack component snapshot test 1`] = `
Array [
  <div
    className="react-reveal"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <div
      className="justify-content-center row"
    >
      <h2
        className="subject-header"
      >
        Tech Stack
      </h2>
    </div>
  </div>,
  <div
    className="react-reveal"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <div
      className="justify-content-center row"
    >
      <div
        className="col-xl-2 col-md-4 col-6"
        style={
          Object {
            "marginBottom": "30px",
          }
        }
      >
        <div
          className="flippy-container"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="flippy-cardContainer-wrapper vertical"
          >
            <div
              className="flippy-cardContainer  "
            >
              <div
                className="flippy-card flippy-front "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <img
                  alt="C#"
                  className="rounded mx-auto d-block"
                  src="csharp.png"
                  style={
                    Object {
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                    }
                  }
                  title="C#"
                />
              </div>
              <div
                className="flippy-card flippy-back "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <p
                  className="lead text-center"
                  style={
                    Object {
                      "marginTop": "10%",
                    }
                  }
                >
                  <a
                    href="https://docs.microsoft.com/en-us/dotnet/csharp/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    C#
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xl-2 col-md-4 col-6"
        style={
          Object {
            "marginBottom": "30px",
          }
        }
      >
        <div
          className="flippy-container"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="flippy-cardContainer-wrapper vertical"
          >
            <div
              className="flippy-cardContainer  "
            >
              <div
                className="flippy-card flippy-front "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <img
                  alt="Bootstrap CSS Framework"
                  className="rounded mx-auto d-block"
                  src="bootstrap.png"
                  style={
                    Object {
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                    }
                  }
                  title="Bootstrap CSS Framework"
                />
              </div>
              <div
                className="flippy-card flippy-back "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <p
                  className="lead text-center"
                  style={
                    Object {
                      "marginTop": "10%",
                    }
                  }
                >
                  <a
                    href="https://getbootstrap.com/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Bootstrap CSS Framework
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xl-2 col-md-4 col-6"
        style={
          Object {
            "marginBottom": "30px",
          }
        }
      >
        <div
          className="flippy-container"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="flippy-cardContainer-wrapper vertical"
          >
            <div
              className="flippy-cardContainer  "
            >
              <div
                className="flippy-card flippy-front "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <img
                  alt="React Framework"
                  className="rounded mx-auto d-block"
                  src="react.png"
                  style={
                    Object {
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                    }
                  }
                  title="React Framework"
                />
              </div>
              <div
                className="flippy-card flippy-back "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <p
                  className="lead text-center"
                  style={
                    Object {
                      "marginTop": "10%",
                    }
                  }
                >
                  <a
                    href="https://reactjs.org/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    React Framework
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xl-2 col-md-4 col-6"
        style={
          Object {
            "marginBottom": "30px",
          }
        }
      >
        <div
          className="flippy-container"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="flippy-cardContainer-wrapper vertical"
          >
            <div
              className="flippy-cardContainer  "
            >
              <div
                className="flippy-card flippy-front "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <img
                  alt="JavaScript"
                  className="rounded mx-auto d-block"
                  src="javascript.png"
                  style={
                    Object {
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                    }
                  }
                  title="JavaScript"
                />
              </div>
              <div
                className="flippy-card flippy-back "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <p
                  className="lead text-center"
                  style={
                    Object {
                      "marginTop": "10%",
                    }
                  }
                >
                  <a
                    href="https://developer.mozilla.org/en-US/docs/Web/JavaScript"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    JavaScript
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xl-2 col-md-4 col-6"
        style={
          Object {
            "marginBottom": "30px",
          }
        }
      >
        <div
          className="flippy-container"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="flippy-cardContainer-wrapper vertical"
          >
            <div
              className="flippy-cardContainer  "
            >
              <div
                className="flippy-card flippy-front "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <img
                  alt="Java"
                  className="rounded mx-auto d-block"
                  src="java.png"
                  style={
                    Object {
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                    }
                  }
                  title="Java"
                />
              </div>
              <div
                className="flippy-card flippy-back "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <p
                  className="lead text-center"
                  style={
                    Object {
                      "marginTop": "10%",
                    }
                  }
                >
                  <a
                    href="https://docs.oracle.com/javase/8/docs/technotes/guides/language/index.html"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Java
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xl-2 col-md-4 col-6"
        style={
          Object {
            "marginBottom": "30px",
          }
        }
      >
        <div
          className="flippy-container"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="flippy-cardContainer-wrapper vertical"
          >
            <div
              className="flippy-cardContainer  "
            >
              <div
                className="flippy-card flippy-front "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <img
                  alt="Python"
                  className="rounded mx-auto d-block"
                  src="python.png"
                  style={
                    Object {
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                    }
                  }
                  title="Python"
                />
              </div>
              <div
                className="flippy-card flippy-back "
                style={
                  Object {
                    "transitionDuration": "0.6s",
                  }
                }
              >
                <p
                  className="lead text-center"
                  style={
                    Object {
                      "marginTop": "10%",
                    }
                  }
                >
                  <a
                    href="https://www.python.org/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Python
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <hr
    className="react-reveal featurette-divider"
    style={
      Object {
        "opacity": undefined,
      }
    }
  />,
]
`;
